test/response/header.js

```diff
@@ -16,5 +16,12 @@ describe('res.header', () => {
       res.res._headers = null;
       res.header.should.eql({});
     });
+
+    it('should use res.getHeaders() accessor when available', () => {
```

Comment:
we should have a test `ctx.response.header` w/o mocking like this because we are testing node v8 in travis. the test should be backported to v1.x as well